docs/Installation.md

```diff
@@ -52,7 +52,7 @@ Uninstallation is documented in the [FAQ](FAQ.md).
 <a name="1"><sup>1</sup></a> For 32-bit or PPC support see
 [Tigerbrew](https://github.com/mistydemeo/tigerbrew).
 
-<a name="2"><sup>2</sup></a> 10.13 or higher is recommended. 10.9–10.12 are
+<a name="2"><sup>2</sup></a> 10.14 or higher is recommended. 10.9–10.13 are
```

Comment:
@MikeMcQuaid hi. so if I understand correctly, this is where you dropped support for 10.13? What does "supported on a best-effort basis" mean? My home-brew installation is no longer working after the latest upgrade, unfortunately. And I am still unable to upgrade from 10.13 (because I still need Aperture). Any advice? Can I go back to the version before this one somehow? I know that some packages probably won't work but that's still better than an entirely broken homebrew installation. Thanks for any advice!

Reply:
(Aperture runs on Mojave, which Apple still supports, and so do we)

"supported on a best-effort basis" means we won't break it deliberately, and if it gets broken and we are provided with a simple fix, we'll review and include it.

Reply:
> this is where you dropped support for 10.13?

Yes.

> "supported on a best-effort basis" means we won't break it deliberately, and if it gets broken and we are provided with a simple fix, we'll review and include it.

Yes but Homebrew maintainers are unlikely to fix problems on 10.13 and binary packages on 10.13 are likely to disappear over time.

Reply:
Thanks guys! This community is awesome. @fxcoudert I had somehow missed that Aperture is also supported on Mojave. I'll move up one minor version for now 😄. That'll probably be easier for me than trying to submit a PR to fix my issue with 10.13.

Keep up the good work!